spec/tapioca/cli_spec.rb

```diff
@@ -451,47 +452,50 @@ def title=(title); end
       assert_path_exists("#{outdir}/namespace/comment.rbi")
       refute_path_exists("#{outdir}/user.rbi")
 
-      assert_equal(<<~CONTENTS.chomp, File.read("#{outdir}/baz/role.rbi"))
+      assert_equal(<<~CONTENTS, File.read("#{outdir}/baz/role.rbi"))
```

Comment:
No need to `chomp` since this PR fixes https://github.com/Shopify/tapioca/issues/346.